passes/memory/memory_bram.cc

```diff
@@ -403,6 +409,7 @@ bool replace_cell(Cell *cell, const rules_t &rules, const rules_t::bram_t &bram,
 	int mem_abits = cell->getParam("\\ABITS").as_int();
 	int mem_width = cell->getParam("\\WIDTH").as_int();
 	// int mem_offset = cell->getParam("\\OFFSET").as_int();
+        bool memory_attribute = cell->attributes["\\ram_style"].decode_string()=="block";
```

Comment:
Is this, and a number of other lookups below, intended to be hard coded to `ram_style`/`block` rather than taking the value from the rules file?

Reply:
The commit I just pushed gets the attribute name from `$mem` cell and compares it to the one defined in the match rule, instead of fixing it to `block`.